src/libs/ajax.js

```diff
@@ -131,12 +131,54 @@ const Leonardo = signal => ({
   }
 })
 
+const workspaceId = getConfig().workspaceId
+const containerId = getConfig().containerResourceId
+const WorkspaceManager = signal => ({
+  /**
+   * Request a SAS token from Workspace Manager.
+   * This SAS token will have permission to view the files in the associated Blob storage container.
+   * @returns {string} A SAS token that may be used for future requests to Azure blob storage.
+   */
+  getSASToken: async () => {
+    const path = `${workspaceId}/resources/controlled/azure/storageContainer/${containerId}/getSasToken?sasExpirationDuration=28800`
```

Comment:
We usually centralize paths/request functions in one place but I don't see a profound need to do so while there still aren't that many.